src/assets/components/ThoughtFeed/ThoughtFeed.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect, useState } from "react";
+import "./ThoughtFeed.css";
+import moment from "moment";
+
+const APIURL = "https://happy-thoughts-ux7hkzgmwa-uc.a.run.app/thoughts";
+
+export const ThoughtFeed = () => {
+  const [recentThoughts, setThoughts] = useState([]);
+  const [likedThoughts, setLikedThoughts] = useState([]);
+
+  useEffect(() => {
+    fetchData();
+  }, []);
+
+  const fetchData = async () => {
+    try {
+      const response = await fetch(APIURL);
+      if (!response.ok) {
+        throw new Error("Network response was not ok");
+      }
+      const json = await response.json();
+      setThoughts(json);
+    } catch (error) {
+      console.error("Error fetching thoughts:", error);
+    }
+  };
+
+  const handleLike = async (thoughtId) => {
+    try {
+      const response = await fetch(`${APIURL}/${thoughtId}/like`, {
```

Comment:
interesting way to change the APIURL to the "liked" version. Cool to see a different approach.